src/zoid/card-fields/component.js

```diff
@@ -122,6 +122,16 @@ export function getCardFieldsComponent() : ZoidComponent<CardProps> {
                     type:       'object',
                     required:   false,
                     queryParam: true
+                },
+
+                'disable-card': {
```

Comment:
Let's use `disableCard` camel casing here for consistency.

Reply:
sounds good to me. Just FYI that JS SDK is having `disable-card` card as a query param.

Reply:
Sure. Normally the convention we use is camelCase for js variable or prop, dash for query param

Reply:
You can still pass `queryParam:    'disable-card',` 👍

Reply:
oh, I see what you're saying now.